src/app/contextBroker/contextBroker.cpp

```diff
@@ -191,7 +191,7 @@
 #include "cache/subCache.h"
 #include "cache/SubscriptionCache.h"
 
-
+using namespace orion;
```

Comment:
I would like to open a discussion about using

    using namespace XXX

instead of writing the namespace out everywhere it is used.

My point of view: If we have more than one type with the same name (we already have Subscription, under different namespaces), it makes it very complicated to locate the use of these types.

A 'grep' of "Subscription" gives us both xxx::Subscription and yyy::Subscription.
This is not very good ...

I vote for keeping our namespaces very short (they all are), and writing them out everywhere. Like that there is no room for confusion.